.github/workflows/build-centos7-assets.yaml

```diff
@@ -0,0 +1,44 @@
+name: Build CentOS 7 Buildbox Assets Images
+run-name: Build CentOS 7 Buildbox Assets Images
+on:
+  # Only allow manual triggers
```

Comment:
Why not also on changes to `build.assets/Dockerfile-centos7-assets`?

Reply:
Because Clang builds take forever, and we probably don't want to update it without knowing about it. If someone decides to fix white spaces in that file, we will trigger the build.